composer.json

```diff
@@ -32,15 +39,15 @@
     "require-dev": {
         "doctrine/orm": "^2.7",
         "myonlinestore/coding-standard": "^3.0",
-        "phpbench/phpbench": "dev-master",
+        "phpbench/phpbench": "^1.1",
         "phpunit/phpunit": "^8.5",
         "psalm/plugin-phpunit": "^0.15",
         "vimeo/psalm": "^4.6"
     },
     "config": {
         "sort-packages": true,
         "platform": {
-            "php": "7.2.24"
+            "php": "7.4.24"
```

Comment:
This was already incorrect before your change, but `platform` has to be removed in order for CI to correctly verify PHP 8.